src/patternfly/layouts/Gallery/gallery.scss

```diff
@@ -1,8 +1,30 @@
 .pf-l-gallery {
   --pf-l-gallery--m-gutter--GridGap: var(--pf-global--gutter);
-  --pf-l-gallery--GridTemplateColumns: repeat(auto-fill, minmax(250px, 1fr)); // The 250px value is here just so that the grid has a width to use in the minmax() for laying out the gallery columns.
+  --pf-l-gallery--GridTemplateColumns--min: 250px;
+  --pf-l-gallery--GridTemplateColumns--min--core: var(--pf-l-gallery--GridTemplateColumns--min);
```

Comment:
Love this change. Though what does `--core` mean here? Maybe we should structure it like the modal's top aligned spacer, and name the overridable thing something user friendly, and that maps to the var name that we build off of the selector/property name. So here, keep `--pf-l-gallery--GridTemplateColumns--min`, and change `--pf-l-gallery--GridTemplateColumns--min--core` to `--pf-l-gallery--GridTemplateColumns--minmax--min`?

Reply:
Love it!